pkg/grpc/authz/user/role.go

```diff
@@ -30,7 +30,7 @@ func (p *roleChecker) Authorized(ctx context.Context, _ string) error {
 
 func (p *roleChecker) checkRole(roleNames []string) error {
 	if len(roleNames) == 0 {
-		return errox.NoValidRole
+		return errox.NotAuthorized
```

Comment:
how about adding more context here e.g. 
```suggestion
		return errox.NotAuthorized.CausedBy("no roles assigned to the user"))
```